.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
 - sudo apt-get update
 - sudo apt-get install -y libapparmor1
 - pecl install -f ast-1.0.3
-- sudo apt-get install npm
+- sudo apt-get install nodejs-dev node-gyp libssl1.0-dev npm
```

Comment:
what are these new dependencies? Are they the same as documented elsewhere in our documentation?

Reply:
I think this is specific to the way Travis preloads things for Bionic. This was required to get node to stall properly (which in turn installs npm). Without this line, the build will fail

Reply:
@kongtiaowang can you try installing LORIS on a new ubuntu 18.04 machine without these added dependencies and see if it works ?

John is saying these changes are needed for travis but we want to know if they are also needed for regular installs

Reply:
@kongtiaowang

Reply:
@kongtiaowang please answer comment above